fabfile/requirements.txt

```diff
@@ -1,26 +1,3 @@
-contextlib2==0.4.0
```

Comment:
As far as I can tell, literally nothing. Potentially we can remove what's in there now and make it the requirements.txt file for Yurt dependencies, so that users don't have to `pip install` all the pre-reqs one at a time.